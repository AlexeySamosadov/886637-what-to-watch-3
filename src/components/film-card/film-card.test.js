import React from "react";
import renderer from "react-test-renderer";
import FilmCard from "./film-card.jsx";

const filmsData = [{
  name: `The Grand Budapest Hotel`,
  date: 2014,
  genre: `Drama`,
  src: `www.rr.i/fsfsdf`,
  id: `55`,
}];

const {name, src} = filmsData[0];
const handleMouseOver = () => {};

it(`This is FilmCard Unit test`, ()=> {
  const tree = renderer.
  create(
      <FilmCard
        src={src}
        handleMouseOver={handleMouseOver}
        name={name}
      />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
