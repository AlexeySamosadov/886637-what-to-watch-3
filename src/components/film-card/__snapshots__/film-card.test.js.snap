// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`This is FilmCard Unit test 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  onClick={[Function]}
>
  <div
    className="small-movie-card__image"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <video
      className="player__video"
      poster="img/somePath"
    />
  </div>
  <h3
    className="small-movie-card__title"
  >
    <a
      className="small-movie-card__link"
      href="movie-page.html"
    >
      The Grand Budapest Hotel
    </a>
  </h3>
</article>
`;
